src/admin/admin.component.tsx

```diff
@@ -156,8 +156,8 @@ function AdminPage() {
         </Grid>
       )}
 
-      {adminFunction === 'units' && <Units />}
-      {adminFunction === 'usage-status'}
+      {adminFunction === 'units'}
```

Comment:
As mentioned on https://github.com/ral-facilities/inventory-management-system/pull/535 there will likely be a period where this is on dev without the units pr so it would be nice if it just displays a work in progress message.

Reply:
I've left the `units.component.tsx` file with just a work in progress message like in #535